exercise/js/main.js

```diff
@@ -39,11 +39,11 @@ $("#button1").on('click', question1);
  * Uncomment the code below and solve the problem
  */
 
-	// function question3(){
-	// 	$("replaceMe").html("replaceMe");
-	// }
+	function question3(){
+	$(".falseFact").html('falseFact');
```

Comment:
` * When button 1 is clicked, select the ordered list item that has the class "falseFact".
 * Using jQuery change the content to read "True Fact".`